sklearn/utils/estimator_checks.py

```diff
@@ -1121,9 +1116,6 @@ def check_classifiers_train(name, classifier_orig):
                     assert_equal(decision.shape, (n_samples, n_classes))
                     assert_array_equal(np.argmax(decision, axis=1), y_pred)
 
-                # raises error on malformed input
-                assert_raises(ValueError,
```

Comment:
can you tell me why this change is necessary?

Reply:
because it's a duplicate of the next two lines